### PR TITLE
Update AWS module dependencies for Wazuh agents

### PR DESCRIPTION
**Pull request by CarlosRS9**
Hello team,

This PR updates the dependencies for using the Azure, Docker and GCP modules in Wazuh agents. The required version for each dependency has been specified. A new documentation page for the Azure and Docker modules dependencies has been created.

To test the required dependencies clean ubuntu and CentOS environments were used. Wazuh agent was installed in both environments and registered in a Wazuh manager. After that, we tried to run each module, resolving any dependency issues one by one.

### CODE REVIEW

File: source/amazon/services/prerequisites/dependencies.rst [hunk 1]
```diff
@@ -35,22 +35,36 @@ Pip
...
 
-b) For **Debian/Ubuntu** systems:
+b. From sources:
+
+.. code-block:: console
+
+  # yum update && yum install curl python3-distutils
+  # curl -O https://bootstrap.pypa.io/get-pip.py
+  # python3 get-pip.py
+
```
Comment (vicferpoy) [inline source/amazon/services/prerequisites/dependencies.rst:53]: Is this necessary? It is pretty much the same but with more steps. Feel free to discuss it here.

File: source/amazon/services/prerequisites/dependencies.rst [hunk 2]
```diff
@@ -35,22 +35,36 @@ Pip
...
   # apt-get update && apt-get install python3-pip
 
-c) From sources:
+b. From sources:
 
 .. code-block:: console
 
+  # apt-get update && apt-get install curl python3-distutils
   # curl -O https://bootstrap.pypa.io/get-pip.py
   # python3 get-pip.py
```
Comment (vicferpoy) [inline source/amazon/services/prerequisites/dependencies.rst:70]: Same here. I understand the first lines were added for consistency but I would like to know if there is a big difference between these two paths.

File: source/amazon/services/prerequisites/dependencies.rst [hunk 3]
```diff
@@ -64,4 +78,4 @@ To install boto3, execute the following command:
 
 .. code-block:: console
 
-  # pip3 install boto3==1.17.76
+  # pip install boto3==1.17.76
```
Comment (vicferpoy) [inline source/amazon/services/prerequisites/dependencies.rst:81]: We assumed that the machine could have different python versions above:

```shellsession
python3 get-pip.py
```

Why did you remove the explicit `pip3` call?

**Review by vicferpoy — CHANGES_REQUESTED** [review]

File: source/amazon/services/prerequisites/dependencies.rst (hunk 1 above)
Comment (CarlosRS9) [reply source/amazon/services/prerequisites/dependencies.rst]: After discussing it with the rest of the team we concluded we should remove this as it is no longer needed.

File: source/amazon/services/prerequisites/dependencies.rst (hunk 3 above)
Comment (CarlosRS9) [reply source/amazon/services/prerequisites/dependencies.rst]: This was restored from pip3 to pip as a mistake. It was set back to `pip3` instead.

**Review by vicferpoy — APPROVED** [review]
LGTM!

**Review by MarVergara — APPROVED** [review]
LGTM. I did some editorial changes to the section to improve consistency and readability.
